### PR TITLE
citra_qt: Add Continue/Pause & Toggle Speed Limit hotkeys

### PR DESCRIPTION
**Pull request by valentinvanelslande**
Default hotkeys:
Continue/Pause: F4
Toggle Speed Limit: CTRL+Z

---
This change is [Reviewable](https://reviewable.io/reviews/citra-emu/citra/3594)

### CODE REVIEW

**Comment by adityaruplaha** [comment]
Excuse me, but I've already made one for Toggle Speed Limit (Y) in adityaruplaha:new-frame-limit-hotkey.

**Comment by jabroany** [comment]
This will be such a huge improvement to the user experience after the current increase in speed.

**Comment by LogPod** [comment]
Could make the default key for Toggle Speed Limit, Tab?

**Comment by jabroany** [comment]
Indeed, Tab would be to prefer compared to Ctrl+z. It'll at least be more in sync with other emulators

File: src/citra_qt/main.cpp [hunk 1]
```diff
@@ -285,11 +285,14 @@ void GMainWindow::InitializeRecentFileMenuActions() {
 void GMainWindow::InitializeHotkeys() {
     RegisterHotkey("Main Window", "Load File", QKeySequence::Open);
     RegisterHotkey("Main Window", "Start Emulation");
+    RegisterHotkey("Main Window", "Continue/Pause", QKeySequence(Qt::Key_F4));
     RegisterHotkey("Main Window", "Swap Screens", QKeySequence(tr("F9")));
     RegisterHotkey("Main Window", "Toggle Screen Layout", QKeySequence(tr("F10")));
     RegisterHotkey("Main Window", "Fullscreen", QKeySequence::FullScreen);
     RegisterHotkey("Main Window", "Exit Fullscreen", QKeySequence(Qt::Key_Escape),
                    Qt::ApplicationShortcut);
+    RegisterHotkey("Main Window", "Toggle Speed Limit", QKeySequence("CTRL+Z"),
```
Comment (adityaruplaha) [inline src/citra_qt/main.cpp]: This is a highly requested feature & I (and many users) will prefer it near to the most used keys. Consider using "Y" (close to D-Pad, iirc) or ";" (close to the arrow keys) by default.

Comment (FearlessTobi) [reply src/citra_qt/main.cpp]: I would really like Tab, because I'm used to it from other emulators, e.g. Dolphin.

**Comment by jroweboy** [comment]
Sorry I've been so negligent about reviewing PRs. :) Thanks for the contribution!

**Pull request merged**